src/main/java/core/basesyntax/db/CommandDaoImpl.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.db;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import model.FruitStorage;
+import model.FruitTransaction;
+import model.FruitTransactionStorage;
+
+public class CommandDaoImpl implements CommandDao {
+    private static final String INPUT_FILE_NAME = "storage.csv";
+    private static final String REPORT_FILE_NAME = "shop_report.csv";
+    private static final String REPORT_FIRST_ROW = "fruit,quantity" + System.lineSeparator();
+    private static final String SPLIT_REGEX = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+
+    @Override
+    public FruitTransactionStorage read() {
+        List<FruitTransaction> transactions = new ArrayList<>();
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(INPUT_FILE_NAME))) {
+            reader.readLine();
+            String line;
+            while ((line = reader.readLine()) != null) {
+                String[] parts = line.split(SPLIT_REGEX);
+                FruitTransaction transaction = new FruitTransaction();
+                transaction
+                        .setOperation(FruitTransaction
```

Comment:
separate your concerns. Create FileReader which only reads, and another service to parse the data into models. Think about validation - what if will get NumberFormatException from .parseInt or enum code won't be found?